src/gtk/toga_gtk/widgets/base.py

```diff
@@ -47,7 +47,8 @@ def set_alignment(self, alignment):
         pass
 
     def set_hidden(self, hidden):
-        self.interface.factory.not_implemented('Widget.set_hidden()')
+        if hidden:
+            self.native.hide()
```

Comment:
What if happens if `not hidden`?

Reply:
For completeness, should add that case as well; although won't it work as it? I looked at [Pack](https://github.com/pybee/toga/blob/9e887767f53d19ed45932678433c17c047f5e08e/src/core/toga/style/pack.py#L91) and figured the inherent property should be not hidden hence didn't pay much attention to it. Either way - completeness is always better. 😄

Reply:
Of course it will work. "set hidden = False" means "make not hidden" - usually rendered as "widget.show()" or similar.

Reply:
Have added the else part..